cdap-docs/faqs/source/hydrator.rst

```diff
@@ -0,0 +1,46 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Frequently Asked Questions about starting the Cask Data Application Platform
+    :copyright: Copyright © 2016 Cask Data, Inc.
+
+:titles-only-global-toc: true
+
+.. _faqs-hydrator:
+
+===================
+FAQs: Cask Hydrator
+===================
+
+What should I do if I see any of these errors in the Cask Hydrator User Interface?
+----------------------------------------------------------------------------------
+- **Error parsing widgets JSON for the plugin <plugin-name>**
+
+  This error means the widget JSON for the particular plugin has an error in its JSON, such as: 
+
+  - You are missing a comma character (``,``) in the JSON
+  - You have a missing a double quote character (``"``) in the JSON
+  - You have a value for a key that is not a valid JSON value (for instance, ``NaN``)
+
+  The fastest way to fix these issues is to run the plugin's Widget JSON through a JSON linter 
```

Comment:
is to use json-lint to identify problems and make sure the widget JSON is valid.